canonical/snapcraft.io
# fixed a wrong number of snaps published results for publishers (snapcrafters, jetbrains, and kde)

## Done
- fixed a wrong number of snaps published results for publishers (snapcrafters, jetbrains, and kde)
 
## How to QA
- Go to https://snapcraft-io-4234.demos.haus/publisher/snapcrafters
- Check if there are 77 snaps published for snapcrafters
- Go to https://snapcraft-io-4234.demos.haus/publisher/jetbrains
- Check if there are 16 snaps published for jetbrains 
- Go to https://snapcraft-io-4234.demos.haus/publisher/kde
- Check if there are 91 snaps published for kde


## Issue / Card
Fixes #https://warthogs.atlassian.net/browse/WD-2769
Fixes https://github.com/canonical/snapcraft.io/issues/4226
